zeeguu/api/endpoints/sessions.py

```diff
@@ -1,11 +1,30 @@
 import flask
+from datetime import datetime
 from flask import request, make_response
 from zeeguu.core.model import Session, User
 from zeeguu.api.utils.abort_handling import make_error
 
-from zeeguu.api.utils.route_wrappers import cross_domain, with_session
+from zeeguu.api.utils.route_wrappers import cross_domain, has_session
 from . import api, db_session
 
+MAX_TIME_SESSION = 30  # Days
+
+
+def validate_session(session_object):
```

Comment:
is what this method does better summarized by `update_session_if_not_expired`? 

although this is also not good enough: the method returns a True/False so this should be clear from it's name.  

I think this method does too many things, and we should split it maybe?

Reply:
So this method is responsible for validating the expire date based on last use of the session. So it checks against the date and if the session is too old it deletes the old session and returns false.

I guess I could not delete it and just return false and the user would have to re-log and create a new uuid, so that delete could be removed. It's just a way of cleaning up.

I have renamed the method `is_session_valid`.